ontologies/ebucore.rdfs.ttl

```diff
@@ -145,6 +145,17 @@ ebucore:hasAnnotationRelatedArtefact a rdf:Property ;
   skos:definition "Pour identifier un objet d'artefact de l'annotation."@fr ;
   skos:definition "Om een artefactonderwerp van de annotatie te identificeren."@nl .
 
+ebucore:hasAnnotationRelatedAgent a rdf:Property ;
```

Comment:
FWIW, you can remove [ebucore:hasAnnotationPurpose](http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#hasAnnotationPurpose), [ebucore:hasAnnotationRelatedAgent](https://www.ebu.ch/metadata/ontologies/ebucore/ebucore_hasAnnotationRelatedAgent.html), and [ebucore:hasAnnotationRelatedLocation](https://www.ebu.ch/metadata/ontologies/ebucore/ebucore_hasAnnotationRelatedLocation.html). They can all be replaced by `ebucore:hasAnnotationAgent`

Reply:
Maybe we should comment them out for now? We do not have any instances yet, but with the future NER data we will probably need them again.